tasks/functions/functions.js

```diff
@@ -51,29 +79,48 @@ function getGoogle(n) {
  *    age: 42
  * }
  */
-function getUser(firstName, lastName, age) {
-
-}
+/*function getUser(firstName = null, lastName = null, age = null) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+  }
+  
+  var person = new getUser("John", "Doe", 42);
+  console.log (person);
+  var person = new getUser("John", "Doe");
+  console.log (person);
+*/
+function getUser(firstName = null, lastName = null, age = null) {
+  var obj = {
+    firstName: firstName,
+    lastName: lastName,
+    age: age
+};
+  return obj;
```

Comment:
you can simply return the object:
...
return {
 firstName,
 lastName,
 age
}

Reply:
done